cnnlabs/cnn-antools-factors-fe
# CNNETANTWO-99: Blacklist Content on Content Details Page

# Why

This PR adds a toggle to the content-details page that puts article on Blacklist for Apple News per epic CNNETANTWO-99 (http://tickets.turner.com/browse/CNNETANTWO-99)

# What

## Fixes:

http://tickets.turner.com/browse/CNNETANTWO-100
http://tickets.turner.com/browse/CNNETANTWO-101
http://tickets.turner.com/browse/CNNETANTWO-102

# How to Test:

1. `git fetch -p`
2. `git checkout CNNETANTWO-99`
3. `git pull`
4. `nvm use`
5. `npm i`
6. `npm start`
7. [http://localhost:5050/content](http://localhost:5050/content)
8. Pick an article
9. Click toggle under _Do Not Publish_ header
10. Click confirm on modal

**Content Details before toggle selected:**
![screen shot 2017-08-23 at 2 23 16 pm](https://user-images.githubusercontent.com/13425204/29632522-3cb9f9c4-8811-11e7-8912-4705fe58475e.png)
**Modal View:**
![screen shot 2017-08-23 at 2 23 35 pm](https://user-images.githubusercontent.com/13425204/29632538-4a15aece-8811-11e7-88bc-b5d926a248b5.png)
**After Blacklist Enabled:**
![screen shot 2017-08-23 at 2 23 44 pm](https://user-images.githubusercontent.com/13425204/29632557-5449c3f8-8811-11e7-9c60-d1d2a8591826.png)
